examples/with-zeit-fetch/fetch/browser.js

```diff
@@ -0,0 +1,3 @@
+import fetch from 'isomorphic-fetch'
```

Comment:
This is supposed to be  `from 'unfetch'`?

https://github.com/zeit/next.js/pull/9029/files#diff-3edae0f8f6e1884bc6c1fa26613acdadR15